src/m365/spfx/commands/project/project-upgrade/rules/FN025001_ESLINTRCJS_overrides.ts

```diff
@@ -0,0 +1,432 @@
+import { Finding } from '../../report-model';
+import { Project } from '../../project-model';
+import { TsRule } from './TsRule';
+import ts = require('typescript');
+
+export class FN025001_ESLINTRCJS_overrides extends TsRule {
+  constructor() {
+    super();
```

Comment:
Since we're not passing any additional information here, we can remove this. I can see we've got this in other TsRules but I don't think we need it

Reply:
Good spot. Removed.